server/app/views/v1/certificates/_certificate.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.id certificate.to_path
```

Comment:
This `id` is misleading... there is (unfortunately) no `.../certificates/:grid/:domain` API that this ID could refer to :(

BTW: The `GridDomainAuthorization#to_path` is even more broken, it uses a non-existent `self.name` field.